apps/prairielearn/src/ee/lib/ai-grading/ai-grading.ts

```diff
@@ -241,6 +243,20 @@ export async function aiGrade({
         rubric_items,
       });
 
+      // If the submission contains images, prompt the model to transcribe any relevant information
+      // out of the image.
+      const explanationDescription = run(() => {
+        const parts = ['Instructor-facing explanation of the grading decision.'];
+        if (hasImage) {
+          parts.push(
+            'You MUST include a complete transcription of all relevant text, numbers, and information from any images the student submitted.',
+            'You MUST transcribe the final answer(s) from the images.',
+            'Use LaTeX formatting for any mathematical expressions.',
```

Comment:
This, plus the changes to `QuestionContainer.tsx`, give us this:

<img width="1524" height="148" alt="Image" src="https://github.com/user-attachments/assets/1e877aca-34e3-4689-a7a6-f05cdd91c702" />